pkg/connector/client.go

```diff
@@ -257,6 +258,11 @@ func (tc *TwitterClient) convertToMatrix(ctx context.Context, portal *bridgev2.P
 		},
 	}
 
+	content := twitterfmt.Parse(ctx, portal, msg)
+	if content != nil {
+		textPart.Content = content
+	}
```

Comment:
It looks like `Parse` always returns a non-nil content, so I don't think there's any point in this `if`? It could just be in the struct literal: `Content: twitterfmt.Parse(ctx, portal, msg)`